src/Lykke.Service.BlockchainWallets.Core/Lykke.Service.BlockchainWallets.Core.csproj

```diff
@@ -8,13 +8,17 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Lykke.Common" Version="6.1.0" />
+    <PackageReference Include="Lykke.Common" Version="6.3.0" />
     <PackageReference Include="Lykke.Service.BlockchainSignFacade.Client" Version="1.1.0" />
     <PackageReference Include="Lykke.SettingsReader" Version="2.2.0" />
-    <PackageReference Include="Lykke.Service.BlockchainApi.Client" Version="17.0.0" />
+    <PackageReference Include="Lykke.Service.BlockchainApi.Client" Version="17.1.1" />
     <PackageReference Include="MessagePack" Version="1.7.3.4" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\contract\Lykke.Service.BlockchainWallets.Contract\Lykke.Service.BlockchainWallets.Contract.csproj" />
+  </ItemGroup>
+
   <ItemGroup>
     <Reference Include="Lykke.Service.BlockchainApi.Client">
       <HintPath>..\..\..\..\..\.nuget\packages\lykke.service.blockchainapi.client\9.0.1\lib\netstandard2.0\Lykke.Service.BlockchainApi.Client.dll</HintPath>
```

Comment:
Is this correct link? I think here should be a link to a project

Reply:
We have a prject reference to the Lykke.Service.BlockchainWallets.Contract, and nuget for the Lykke.Service.BlockchainApi.Client. It seems, that everything is ok

Reply:
Here is a version in the path (.client\9.0.1\) after nuget updating the project will not able to find a new dll. Something wrong here.

Reply:
It's only a hint path, in fact, it's optional for visual studio